doc/api.yml

```diff
@@ -1687,6 +1687,117 @@ paths:
           $ref: '#/components/responses/429'
         '500':
           $ref: '#/components/responses/500'
+  '/teachers/{teacher_id}/picture/upload':
+    post:
+      tags:
+        - Users
+      summary: Upload teacher profile picture
+      operationId: uploadTeacherProfile
+      parameters:
+        - name: teacher_id
+          in: path
+          required: true
+          schema:
+            type: string
+          description: Teacher id to upload profile picture
```

Comment:
owner of the profile picture is simpler but i do not think you need a description at all.